code-challenges.test.js

```diff
@@ -37,23 +63,89 @@ var randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    it("logs decides if the number is evenly divisble by three or not", () => {
+  
+      //an expect method, nested within the test block, calling on the capitalizer() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(capitalizer(array1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+      expect(capitalizer(array2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+    })
+  })
 
 // b) Create the function that makes the test pass.
 
-
-
+  //declare a function that takes in an array
+  const capitalizer = (array) => {
+    // create a local variable that maps through the array
+    let eachItem = array.map(value => {
+      // with each value access the first index and capitalize the letter then concatenate the remainder of the word
+      return value[0].toUpperCase() + value.substring(1)
+    })
+    // return the local variable
+    return eachItem
+  }
+  //logging the function and pass in the array
+  console.log(capitalizer(randomNouns1));
+
+//-------------------------------------------------------------------------------------------------------------------------------------
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
-// a) Create a test with expect statements for each of the variables provided.
-
 var vowelTester1 = "learn"
 // Expected output: 1
 var vowelTester2 = "academy"
 // Expected output: 0
 var vowelTester3 = "challenges"
 // Expected output: 2
 
+//  a) Create a test with expect statements for each of the variables provided.
+
+// describe method that lists the name of the function OR naming of the particular test.
+  describe("firstVowellocator", () => {
+    // redeclare the arrays in the test
+    let string1 = "learn"
+    let string2 = "academy"
+    let string3 = "challenges"
+    // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+    it("logs the index of the first vowel", () => {
+  
+      //an expect method, nested within the test block, calling on the firstVowellocator() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+      expect(firstVowellocator(string1)).toEqual(1)
+      expect(firstVowellocator(string2)).toEqual(0)
+      expect(firstVowellocator(string3)).toEqual(2)
+    })
+  })
+
 
 
 // b) Create the function that makes the test pass.
+
+
+// declare a function that takes in an string
+// const firstVowellocator = (string) => {
+       // my thought is to split the string into a single letter and store in an array     
+//     let array = string.slipt("") // But when I run this code by using "yarn jest", it shows TypeError: string.slipt is not a function
+//     // return a higher-order function that takes in the value and the index
```

Comment:
Looks like just a spelling error on the method. `string.slipt` should be `string.split`